grouper/permissions.py

```diff
@@ -61,6 +62,39 @@ def grant_permission(session, group_id, permission_id, argument=''):
     session.commit()
 
 
+def grant_permission_to_tag(session, tag_id, permission_id, argument=''):
+    # type: Session, int, int, str -> None
```

Comment:
are these `type:` declarations meant to conform to [PEP 484](https://www.python.org/dev/peps/pep-0484/#id44)? if i'm not mistaken these will all fail things like `mypy` because
1. they reference objects that are not imported, `Session` in this case and
2. the formatting is off, should be something like `# type: (Session, int, int, str) -> None`

I'm happy you're adding these but if can run these through `mypy` to make sure they work it'd help when we add `mypy` to CI.